crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -262,6 +262,7 @@ def status
                 "pending" => 0,
                 "unready" => 0,
                 "building" => 0,
+                "crowbar-upgrade" => 0,
```

Comment:
BTW, as this is anyway just for displaying on the frontend even other upgrade states can be grouped within that.